### ISSUE TITLE
Does azure IoT hub supports LwM2M protcol ?

### ISSUE THREAD

**Issue by kanumuru** [Azure/azure-iot-protocol-gateway#155]
Hi Team,
Does azure IoT hub support LwM2M protocol?
Any information related to this is helpful

**Comment by moravcik94** [comment]
I don't think so. They've implemented device management by encapsulation of MQTT protocol.
